properties.py

```diff
@@ -264,6 +264,16 @@ def update_type(self, context):
         name="AOV name",
         description="Name of the built in AOV")
 
+    denoise_aov = BoolProperty(
+        name="Format AOV for denoising",
+        description="If checked this pass will be export as an individual file and properly formatted for use by the denoise utility",
+        default=False)
+
+    exclude_from_multi = BoolProperty(
```

Comment:
i didn't have a particular example in mind but I thought it would give a user more flexibility on how their passes and images are exported.